client/common/utils.tsx

```diff
@@ -964,6 +964,10 @@ export const statCalculators: { [key: string]: StatCalculator } = {
       ? getStatWithDefault(statsFromCustomSet, Stat.DAMAGE) +
         getStatWithDefault(statsFromCustomSet, Stat.AIR_DAMAGE)
       : 0,
+  [Stat.PODS]: (statsFromCustomSet) =>
+    100 +

```

Comment:
Are you sure about this? I thought new characters started with like 1k pods?

Reply:
You are right. It's a typo probably